code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -104,6 +105,61 @@
 	desc = "A sleek, sturdy box"
 	icon_state = "box_of_doom"
 
+/*New Uplink Bundles*/
+/obj/item/weapon/storage/box/syndie_kit/darklord/New() //Now you too can play as darth vader
+	..()
+	new /obj/item/weapon/melee/energy/sword/saber/red(src)
+	new /obj/item/weapon/melee/energy/sword/saber/red(src)
+	new /obj/item/weapon/dnainjector/telemut/darkbundle(src)
+	new /obj/item/clothing/suit/hooded/chaplain_hoodie(src)
+	new /obj/item/weapon/card/id/syndicate(src)
+	new /obj/item/clothing/mask/chameleon(src)
+
+/obj/item/weapon/storage/box/syndie_kit/sleepingcarp/New() //Space Ninjas aren't just admin only anymore.
+	..()
+	new /obj/item/weapon/storage/box/syndie_kit/throwing_weapons(src)
+	new /obj/item/weapon/sleeping_carp_scroll(src)
+	new /obj/item/weapon/twohanded/bostaff(src)
+	new /obj/item/weapon/card/id/syndicate(src)
+	new /obj/item/clothing/under/chameleon(src)
+	new /obj/item/clothing/head/chameleon(src)
+	new /obj/item/clothing/mask/chameleon(src)
+	new /obj/item/weapon/grenade/smokebomb(src)
+
+/obj/item/weapon/storage/box/syndie_kit/awperative/New() //SYNDIES CALL THE SHUTTLE!
+	..()
+	new /obj/item/clothing/suit/space/hardsuit/syndi(src)
+	new /obj/item/weapon/implanter/explosive(src)
+	new /obj/item/weapon/implanter/weapons_auth(src)
+	new /obj/item/weapon/gun/projectile/automatic/c20r(src)
+	new /obj/item/clothing/mask/gas/syndicate(src)
+	new /obj/item/weapon/card/id/syndicate(src)
+	new /obj/item/weapon/c4(src) //Like a cherry on top.
+
+
+/obj/item/weapon/storage/box/syndie_kit/implantbundle/New() //Makes you slippery like a fish. Worse case scenario, just *deathgasp
+	..()
+	new /obj/item/weapon/implanter/freedom(src)
+	//new /obj/item/weapon/implanter/uplink(src)
+	new /obj/item/weapon/implanter/emp(src)
+	new /obj/item/weapon/implanter/adrenalin(src)
+	new /obj/item/weapon/implanter/explosive(src)
+	new /obj/item/weapon/implanter/storage(src)

```

Comment:
Security will never catch you, and if they do, you can get away. And if they kill you, they still get fucked.